types/logs_test.go

```diff
@@ -0,0 +1,403 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestStatusCodesSet(t *testing.T) {
+	tests := []struct {
+		desc     string
+		value    string
+		expected StatusCodes
+	}{
+		{
+			desc:     "One value should return StatusCodes of size 1",
```

Comment:
I think the "should return StatusCodes ..." stuff can be dropped from the test descriptions.

Reply:
I don't agree with that.

Reply:
I'm not super hot about this one, I just think it's good to decrease optical "noise" and redundant explanations.